src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/ActivityServiceImplementation.java

```diff
@@ -0,0 +1,58 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.Activity;
+import ro.societateahermes.backendservice.entities.DTO.ActivityDTO;
+import ro.societateahermes.backendservice.entities.User;
+import ro.societateahermes.backendservice.repository.ActivityRepositoryInterface;
+import ro.societateahermes.backendservice.service.ActivityServiceInterface;
+
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class ActivityServiceImplementation implements ActivityServiceInterface {
+    private final ActivityRepositoryInterface activityRepository;
+
+    public ActivityServiceImplementation(ActivityRepositoryInterface activityRepository) {
+        this.activityRepository = activityRepository;
+    }
+
+    @Override
+    public void save(Activity activity) {
+        activityRepository.save(activity);
+    }
+
+    @Override
+    public List<Activity> getAllActivities() {
+        return activityRepository.findAll();
+    }
+
+    @Override
+    public void delete(Long id) {
+        for(Activity activity  : activityRepository.findAll())
+        {
+            if (activity.getIdActivity() == id ){
+                activityRepository.delete(activity);
+                return;
+            }
+        }
+    }
+
+    @Override
```

Comment:
Foloseste @Transactional deasupra metodei. Altfel modificarile pe care le faci aici nu o sa se reflecte in baza de date